scipy/sparse/base.py

```diff
@@ -537,8 +557,32 @@ def __getattr__(self, attr):
         else:
             raise AttributeError(attr + " not found")
 
-    def transpose(self):
-        return self.tocsr().transpose()
+    def transpose(self, axes=None, copy=False):
+        """
+        Reverses the dimensions of the sparse matrix.
+
+        Parameters
+        ----------
+        axes : None, optional
+            This argument is in the signature *solely* for NumPy
+            compatibility reasons, as reversing the dimensions is
+            the only logical permutation that should be requested.
```

Comment:
Eh, I guess it's not a big deal. Out of curiosity, what are the 13 valid tuples? I would expect `(0,1)` and `(1,0)` to work, but that's about it. Unless we're including negative indices here, but those could be dealt with first in the same way that they're handled elsewhere.